packages/app-content-pages/next.config.js

```diff
@@ -22,12 +18,10 @@ const webpackConfig = require('./webpack.config')
 const SENTRY_CONTENT_DSN = 'https://1f0126a750244108be76957b989081e8@sentry.io/1492498'
 const APP_ENV = process.env.APP_ENV || 'development'
 const COMMIT_ID = process.env.COMMIT_ID || commitID()
-const assetPrefix = assetPrefixes[APP_ENV]
```

Comment:
I’d leave this in, in case we need to load images from an external origin in future. It will still work as long as `assetPrefixes` is an empty object.